packages/graphql-hooks/src/useClientRequest.ts

```diff
@@ -82,18 +100,25 @@ function useClientRequest(query, initialOpts = {}) {
 
   const contextClient = React.useContext(ClientContext)
   const client = initialOpts.client || contextClient
+
+  if (client === null) {
```

Comment:
what if it's undefined?

Reply:
I believe it can't be, if `initialOpts.client` is undefined here it will instead fall to `contextClient` which can either be a client or `null`.